src/install.rst

```diff
@@ -225,6 +225,19 @@ These instructions will install the Nextstrain CLI and tools to run and view you
 
             Setting default environment to <option>.
 
+      .. hint::
+
+         By default, your Windows home directory will not be directly accessible under your WSL home directory. When run in a WSL prompt, the following command fixes that by creating a symlink to it in your WSL home directory. This allows you to use Windows-based text editors and Linux commands all on the same files. Before running,
+
+         1. Replace ``<user>`` with your Windows username.
+         2. Optionally, change the ``windows_home`` folder name or only link to a specific directory under your windows user (e.g. ``ln -s /mnt/c/Users/<user>/Documents ~/windows_documents``).
+
+         .. code-block:: bash
+
+               ln -ws /mnt/c/Users/<user> ~/windows_home
```

Comment:
Does WSL expose the typical Windows environment variables like `USERPROFILE` or `HOMEDRIVE` and `HOMEPATH`? If so, those could be used in this command to avoid the manual "replace X with Y" steps.

Reply:
Oh good thinking! [From StackExchange](https://superuser.com/a/1568668):

```sh
wslpath "$(wslvar USERPROFILE)"
```

I'll make another PR using this.